packages/polyfills/src/base.ts

```diff
@@ -1 +1,2 @@
-require('@babel/polyfill');
+require('core-js/stable');
+require('regenerator-runtime/runtime');
```

Comment:
Just to confirm, this is removed by `@babel/preset-env` when generators are supported?

Reply:
I haven't tested, but it should yes.

`@babel/polyfill` prior to 7.4 was already importing it (and a couple of other things): https://github.com/babel/babel/blob/v7.3.5/packages/babel-polyfill/src/index.js and according to the documentation, it is equivalent to the 2 imports I added.
https://babeljs.io/docs/en/next/babel-polyfill.html